### PR TITLE
Fixed MockDHTProvider

### PR DESCRIPTION
**Pull request by egbertbouman**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,6 @@
     extras_require={
         'netifaces': ["netifaces"],
         'asynctest': ["asynctest"],
-        'marshmallow': ["marshmallow"],
```
Comment (qstokkink) [inline setup.py:28]: This should appear in the `install_requires` then (as `"marshmallow"`).

Comment (egbertbouman) [reply setup.py:28]: Oops! It's fixed now.

**Comment by qstokkink** [comment]
Seems like we overloaded the Jenkins webhook again:
afbeelding

**Comment by qstokkink** [comment]
retest this please

**Comment by qstokkink** [comment]
retest this please

**Pull request merged**